path/windows/resolve.ts

```diff
@@ -32,7 +32,14 @@ export function resolve(...pathSegments: string[]): string {
       ) {
         throw new TypeError("Resolved a relative path without a CWD.");
       }
-      path = Deno.cwd();
+      try {
+        path = Deno.cwd();
+      } catch (error) {
+        if (!(error instanceof Deno.errors.PermissionDenied)) {
+          throw error;
+        }
+        path = undefined as unknown as string;
+      }
```

Comment:
`resolve("c:")` now resolves to `c:\` instead of throwing permission error. I think this is a breaking change to `windows/resolve`

Can we avoid changing `resolve` here, but only making changes to expandGlob?

Reply:
Seems like using directly the root if `isGlobAbsolute` is true instead of `resolve(root)` do the trick